src/HornedBeasts.js

```diff
@@ -0,0 +1,11 @@
+import React, { Component } from 'react'
+
+class HornedBeast extends Component {
+  render() {
+    return (
+        <p>Horned Beast {this.props.num}</p>
+    )
+  }
+}

```

Comment:
Good work so far on lab!

You got the job done but strayed from lab instructions a bit. This component needed:

- to contain an <h2> that displays the title of the animal, an <img> element with src, alt and title attributes, and a <p> that displays the description.